src/parser/jobs/rdm/about.module.css

```diff
@@ -0,0 +1,9 @@
+.bad {
```

Comment:
There's a few colours defined in `src/components/App.css` - pls2 use those. If it's missing one you need, drop a line in deeskerd. (Need to keep consolidated to aid in theming down the road).